.travis.yml

```diff
@@ -110,6 +110,15 @@ jobs:
     - BENCHMARK_REPEATS=1 make benchmark-all
     after_success: skip
 
+  - stage: test
+    python: 3.7
+    name: 'Pydantic-Only Benchmarks'
+    script:
+    - pip install -r benchmarks/requirements.txt
```

Comment:
surely the point of this benchmark is to skip if line to check pydantic only benchmarks work **without** attrs being installed?

Reply:
This was copied directly from the benchmarks step above. Yes, should be removed for obvious reasons.